zap/src/main/java/org/zaproxy/zap/scan/filters/GenericFilterBean.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2019 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.scan.filters;
+
+import java.util.Collection;
+import java.util.LinkedHashSet;
+import org.apache.commons.collections.CollectionUtils;
+
+/**
+ * Generic Filter Bean which can be applied to any Class Object which implements hashCode and equals
+ *
+ * @author KSASAN preetkaran20@gmail.com
+ * @param <T>
```

Comment:
Complete the jdoc here?

Reply:
Didn't got your point. can you please elaborate.

Reply:
Should include a description or expected value of the param(s).

ex: https://www.oracle.com/technetwork/articles/java/index-137868.html#writingdoccomments